### ISSUE TITLE
Reset Molecule button not working

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#305]
@jessegreenberg, the Reset Molecule button is not currently operational with mouse or keyboard. Please have a look.

**Comment by terracoda** [comment]
I looks pressed, so you might have the pressed states reversed.

**Comment by terracoda** [comment]
Actually, there seems to be a lot issues with the other buttons. Some I can operate with mouse and some I can't.

It seems I can't use the mouse on the Play, Pause, or Step Forward buttons, and the Global Sound button is also not operable by mouse. Something weird is going on with the buttons.

**Comment by jessegreenberg** [comment]
I just pulled master and I am not seeing any issues with buttons, including the "Reset Molecule" button. @terracoda what version are you testing? If you open the developer tools do you see any errors?

**Comment by terracoda** [comment]
I'll pull master. I haven't pulled since this morning.

**Comment by terracoda** [comment]
Still having issues...
Could be performance related.
The Reset Molecule didn't work with mouse or keyboard.
The other buttons worked for a bit, then stopped working and Voice Over was saying, "Safari Busy"

No buttons were working once Safari became busy.

**Comment by terracoda** [comment]
Using MacOs 10.13.6, Voice Over and Safari

**Comment by jessegreenberg** [comment]
THanks @terracoda OK ill take a look at that platform

**Comment by jessegreenberg** [comment]
Using macOS 10.15 with and without VoiceOver I am still not seeing this issue. @terracoda is this still happening for you?

**Comment by terracoda** [comment]
Yep. Here's a screen shot of  a pressed button and no molecule:
<img width="912" alt="Screen Shot 2020-03-23 at 10 22 53 PM" src="https://user-images.githubusercontent.com/1902116/77377104-221a0600-6d55-11ea-9e17-0b55bda2d777.png">

**Comment by jessegreenberg** [comment]
OK should be fixed in the above commit, @terracoda can you please make sure I got it?

**Comment by terracoda** [comment]
I verfied the fix @jessegreenberg. Nice work! Closing.
